src/main/scala/amailp/intellij/robot/psi/reference/PythonKeywordToDefinitionReference.scala

```diff
@@ -15,6 +15,13 @@ import com.jetbrains.python.psi.{PyClass, PyFile, PyFunction}
 import scala.collection.JavaConversions._
 import scala.language.reflectiveCalls
 
+class KeywordReference(reference: String) {
+  private val (libraryName, keywordName) = reference.splitAt(reference.lastIndexOf('.'))
```

Comment:
This logic would become easier if you use `QualifiedName.fromDottedString`

Reply:
Now it looks better. Is it possible to do even better? :)